### PR TITLE
Fix invalid size issue while provisioning volume

### PR DESCRIPTION
**Pull request by utkarshmani1997**
1. Why is this change necessary ?
- volume provisioning was failing for Gi,Pi,Mi..etc notations.

2. How does this change address the issue ?
- Added a regular expression which is used to match volume size,
  before provisioning volume in standard format.It will throw error
  if size will not have valid suffix.The standard followed is given
  below:
  - 1 GB = 1000 MB
  - 1 Gi = 1024 MiB
  - ....etc

3. How to verify this change ?
- Run the jiva binary locally or in k8s cluster, it should not fail
  for exp: `./jiva replica --size 1Gi /tmp`

4. What side effects does this change have ?
- none
- Need to handle these cases in maya also, currently mayactl supports volume
  creation in G only.So if you are creating volumes manually from mayactl
  then that will throw you suggestions to use G.

5. Other details
fix: openebs/openebs#540

### CODE REVIEW

File: app/replica.go [hunk 1]
```diff
@@ -92,6 +95,11 @@ checkagain:
 }
 
 func startReplica(c *cli.Context) error {
+	var (
+		volSize            int64
+		isValidSizeDecimal = regexp.MustCompile(`^(\d+(\.\d+)*) ?([kKmMgGtTpP])?[bB]?$`)
```
Comment (AmitKumarDas) [inline app/replica.go]: Can we have these expressions outside of function s.t. it can be unit tested.

Comment (AmitKumarDas) [reply app/replica.go]: Can you write unit tests for these regexes

Comment (utkarshmani1997) [reply app/replica.go]: @AmitKumarDas sure

**Comment by utkarshmani1997** [comment]
This is not required for now, so closing this PR.

**Pull request closed without merging**